nutclient compilation warning

Hello all,

@aquette point me the following compilation warning:

```
../../../clients/nutclient.cpp:1310:3: warning: delete called on 'nut::Client' that is abstract but has non-virtual destructor [-Wdelete-non-virtual-dtor]
                delete (nut::Client*)client;
                ^
1 warning generated.
```

--- a/clients/nutclient.h
+++ b/clients/nutclient.h
@@ -122,7 +122,7 @@ class Client
 	friend class Variable;
 	friend class Command;
 public:
-	~Client();
+	virtual ~Client();
 
 	/**
 	 * Intend to authenticate to a NUTD server.

